tiled/server/app.py

```diff
@@ -111,7 +111,7 @@ def build_app(
     deserialization_registry: Optional[SerializationRegistry] = None,
     compression_registry: Optional[CompressionRegistry] = None,
     validation_registry: Optional[ValidationRegistry] = None,
-    tasks=None,
+    tasks: Optional[dict[str, list[Any]]] = None,
```

Comment:
```suggestion
    tasks: Optional[dict[str, list[Awaitable]]] = None,
```

Shouldn't this be required, as we await the items in the list?

Reply:
I think they should be callables that return awaitables but I was being lazy. Have now updated (here and in the typeddict)